pytest_postgresql/retry.py

```diff
@@ -19,7 +19,7 @@ def retry(
     ... ::
         FATAL:  the database system is starting up
     """
-    time: datetime = datetime.utcnow()
+    time: datetime = datetime.now(UTC)
```

Comment:
This change is not compatible with python 3.8 (eol at October 2024), 3.9 (eol at October 2025) and 3.10 (eol at October 2026).
I'd want to have some compatibility layer for that.

Reply:
Done!